src/services/pact-verifier-service.test.ts

```diff
@@ -37,40 +38,23 @@ describe('PactVerifierService', () => {
     }).compile();
 
     pactVerifierService = moduleRef.get<PactVerifierService>(PactVerifierService);
-    pactVerifierProvider = moduleRef.get(PactModuleProviders.PactVerifier);
+  });
+
+  beforeEach(() => {
+    jest.clearAllMocks();
   });
 
   describe('when configuring the providerBaseHost', () => {
     beforeEach(async () => {
-      options = { ...options, providerHost: '0.0.0.0' };
-      await pactVerifierService.verify(appMock as any);
+      options = { ...options, providerBaseUrl: 'http://127.0.0.1:80' };
+      await pactVerifierService.verify(appMock as unknown as INestApplication);
     });
     test('allows configuring hosts', async () => {
-      expect(appMock.listen).toHaveBeenCalledWith(80, '0.0.0.0');
-    });
-  });
-
-  describe("When calling the 'verify' method", () => {
-    beforeAll(async () => {
-      await pactVerifierService.verify(appMock as any);
-    });
-
-    test("then call 'verifyProvider' with the exact options", async () => {
-      expect(pactVerifierProvider.verifyProvider).toHaveBeenCalledWith({
-        ...options,
-        providerBaseUrl: '127.0.0.1:80',
-      });
+      expect(appMock.listen).toHaveBeenCalledWith(80, '127.0.0.1');
```

Comment:
Am I right in that there's a test that was removed here? What's the reason it was removed?

Reply:
2 tests I think

Reply:
Looks good. I will let @TimothyJones resolve the comment.

Reply:
I don't have a resolve button for some reason, but I'm happy with this.